x-pack/auditbeat/module/system/process/_meta/data.json

```diff
@@ -0,0 +1,29 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "agent": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
+    },
+    "event": {
+        "action": "existing_process",
+        "dataset": "process",
+        "id": "5795d53b-f7c2-463c-9c04-f316ae876d51",
+        "module": "system",
+        "kind": "state"
+    },
+    "message": "Process zsh (PID: 2363) is RUNNING",
+    "process": {
+        "args": [
+            "/usr/bin/zsh"
+        ],
+        "executable": "/bin/zsh",
+        "name": "zsh",
+        "pid": 2363,
+        "ppid": 2362,
+        "start": "2018-12-10T16:36:25.21Z",
+        "working_directory": "/home/elastic"
+    },
+    "service": {
+        "type": "system"
```

Comment:
We can also add this to a follow-up PR for mapping to ECS. But `service.type` is meant to describe the name of a concrete application being monitored by the agent. The fact that this is a system monitoring event is already captured by `event.module:system`.

Reply:
So this [comes from the Metricbeat framework](https://github.com/elastic/beats/blob/master/metricbeat/mb/event.go#L70) we're using in Auditbeat. I think it's set for all of Metricbeat and Auditbeat.

I think we should address it separately and more broadly - I assume all other metricsets have an `event.module` as well (e.g. `event.module: mysql`) that already says what it's monitoring.

Reply:
You're right about `event.module`: #8879